### ISSUE TITLE
Invisible meshes when using optimize

### ISSUE THREAD

**Issue by Sebastian-81** [ousnius/nifly#51]
When using `NifSaveOptions.optimize=true` and saving `cavegewallstraight01.nif ` from  Skyrim Remastered - Caves, the mesh gets invisible:  [Mesh invisible](https://i.imgur.com/ayHRCpt.png) . The mesh is not invisible when `NifSaveOptions.optimize` is set to `false`

The major difference to to original mesh I found was a different bounding sphere

**Comment by ousnius** [comment]
@Sebastian-81 @hakasapl Yes, the calculation of bounding spheres can be "too good" (too small sphere) in certain cases for certain meshes. It's also possible that some meshes have custom bounding spheres, so recalculating them could cause object culling issues.

Feel free to save NIFs without the optimize option. If needed, call DeleteUnreferencedBlocks manually. There's nothing I can do other than provide more granular options for the save function (but you would still need to set them accordingly).

Also feel free to make a PR on nifly for those options if you need/want them.

**Comment by ousnius** [comment]
SSE NIF Optimizer has been using the following options:
```cpp
NifSaveOptions saveOptions;
saveOptions.optimize = false;
saveOptions.sortBlocks = false;
```
